cmd2/cmd2.py

```diff
@@ -280,6 +281,24 @@ def cmd_wrapper(cmd2_instance, statement: Union[Statement, str]):
     return arg_decorator
 
 
+class _SavedReadlineSettings:
+    """readline settings that are backed up when switching between readline environments"""
+    def __init__(self):
+        self.completer = None
+        self.delims = ''
+        self.basic_quotes = None
+
+
+class _SavedCmd2Env:
```

Comment:
Should this be in either utils.py or in pyscript_bridge.py?

Reply:
Since this class is only used in cmd2.py, I'm OK leaving it.